R/modeltests.R

```diff
@@ -203,6 +203,9 @@ evaluateRuns <- function(model, mydir, gitPath, compScen, email, mattermostToken
   write(paste0("This is the result of the automated ", model, " testing suite on ",
                format(Sys.time(), "%Y-%m-%d"), "."), myfile, append = TRUE)
   write(paste0("Path to runs:", mydir, "output/"), myfile, append = TRUE)
+  if (model == "REMIND") {
+    write(paste0(c("Responsibilities:","  Robert     : SSP2EU-EU21", "  Jess / Olli: SSP2EU", "  Bjoern     : SDP"), collapse = "\n"), myfile, append = TRUE)
```

Comment:
nice, thanks, maybe check with those people if they agree, and also add SSP1 and SSP5 so that it get obvious that we still need people feeling responsible for those scenarios

Reply:
Checked with people and added empty SSP[1,5]